### DISCUSSION TITLE
thanhle.blog/blog/optimize-tti-v%C3%A0-fid-cho-nextjs-mot-cach-super-don-gian

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#45] [Ideas]
# thanhle.blog/blog/optimize-tti-v%C3%A0-fid-cho-nextjs-mot-cach-super-don-gian

https://thanhle.blog/blog/optimize-tti-v%C3%A0-fid-cho-nextjs-mot-cach-super-don-gian

**Comment by nguyenhoaikhang37** [comment]
đã star ạ, nào làm prj lq đến nextjs nhất định em sẽ xài npm này, cảm ơn anh

**Reply by thanhlmm** [reply]
Thanks em. Nhớ nha :))))
